tests/test_config.py

```diff
@@ -115,25 +115,25 @@ def test_custom_dirs(tmp_path):
         scene = MyScene()
         scene.render()
 
-        assert_dir_filled(os.path.join(tmp_path, "test_sections"))
-        assert_file_exists(os.path.join(tmp_path, "test_sections", "MyScene.json"))
+        assert_dir_filled(Path(tmp_path).joinpath("test_sections"))
+        assert_file_exists(Path(tmp_path).joinpath("test_sections", "MyScene.json"))
 
-        assert_dir_filled(os.path.join(tmp_path, "test_video"))
-        assert_file_exists(os.path.join(tmp_path, "test_video", "MyScene.mp4"))
+        assert_dir_filled(Path(tmp_path).joinpath("test_video"))
+        assert_file_exists(Path(tmp_path).joinpath("test_video", "MyScene.mp4"))
 
-        assert_dir_filled(os.path.join(tmp_path, "test_partial_movie_dir"))
+        assert_dir_filled(Path(tmp_path).joinpath("test_partial_movie_dir"))
         assert_file_exists(
-            os.path.join(
-                tmp_path, "test_partial_movie_dir", "partial_movie_file_list.txt"
+            Path(tmp_path).joinpath(
+                "test_partial_movie_dir", "partial_movie_file_list.txt"
             )
         )
 
         # TODO: another example with image output would be nice
-        assert_dir_exists(os.path.join(tmp_path, "test_images"))
+        assert_dir_exists(Path(tmp_path).joinpath("test_images"))
```

Comment:
There is an easier way to join paths in pathlib with the "/" operator: https://docs.python.org/3/library/pathlib.html#operators.
`Path(tmp_path).joinpath("test_images")` would then become `tmp_path / "test_images"`
Also, I suggest that you first make tmp_path a pathlib object, and then you can use  tmp_path directly, and you don't need to use Path(tmp_path) anymore

Reply:
Updated all instances of joinpath to "/"